.coveragerc

```diff
@@ -0,0 +1,10 @@
+[run]
+source = src
+
+omit =
+    */phue.py
+    */rgbxy/__init__.py
+    */mock/mock.py
+    */mock/__init__.py
+    */pbr/*.py
+    */six.py
```

Comment:
FYI: here is the generic coveragerc that codecov suggests: https://gist.github.com/codecov-io/bf15bde2c7db1a011b6e
I am not seeing any coverage data for your branch on:
https://codecov.io/gh/google/aiyprojects-raspbian/commits